hack/community/generate.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Given the current WMCO manifests, generate new community manifests.
+
+# Must be ran within WMCO root.
+
+# Example:
+# Cut community v5.1.0 for community-4.10
+# Run: bash ./hack/community/generate.sh 5.1.0 community-4.10
+
+# clone community-operators-prod repo
+clone_community() {
+  local CO_GIT="https://github.com/${CO_REPO}.git"
+
+  git clone "${CO_FORK}" "${CO_DIR}"
+  pushd "${CO_DIR}"
+  git remote add upstream "${CO_GIT}"
+  git fetch upstream main:upstream/main
+  git checkout upstream/main
+  git config user.name "${GITHUB_NAME}"
+  git config user.email "${GITHUB_EMAIL}"
+  echo "   ${CO_PROJECT} cloned"
+  popd
+}
+
+# replace necessary fields with the yq tool
+replace() {
+    export CREATED_AT=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    export VALUE="community-windows-machine-config-operator"
+    export NAME="${VALUE}-${OPERATOR_VERSION}"
+    export CO_DESCRIPTION=${DESCRIPTION}
```

Comment:
Given the `replace()` funcion is invoked with parameters:

`replace "$CO_CSV" "$OPERATOR_VERSION" "$DESCRIPTION" "$QUAY_IMAGE"`

I feel you need to set the value of the variables (i.e. `CO_DESCRIPTION`) using positional arguments. For example:
`export CO_DESCRIPTION=$3`